canonical-web-and-design/vanilla-framework
# Improve README

## Done

Restructured and fleshed out the various READMEs and related markdown docs this project has, hopefully to make the information easier to find and navigate.

Fixes #1020

## QA

- Go to [this branch](https://github.com/sowasred2012/vanilla-framework/tree/project-docs-update)
- Look at the README and see that it guides you towards a lot more developer information, and that it makes sense/is appropriate.

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
